biz.aQute.bndlib/src/aQute/bnd/osgi/Jar.java

```diff
@@ -294,7 +294,10 @@ private Jar buildFromInputStream(InputStream in, long lastModified) throws IOExc
 				int size = (int) entry.getSize();
 				try (ByteBufferOutputStream bbos = new ByteBufferOutputStream((size == -1) ? BUFFER_SIZE : size + 1)) {
 					bbos.write(jin);
-					putResource(entry.getName(), new EmbeddedResource(bbos.toByteBuffer(), lastModified), true);
+					long time = entry.getTime();
```

Comment:
Should this be `long time = ZipUtil.getModifiedTime(entry)`? We seem to prefer that in other places to `ZipEntry.getTime()`.